connectd/usr/bin/connectd_library

```diff
@@ -10,9 +10,9 @@
 #  Copyright 2019. All rights reserved.
 
 ##### Settings #####
-LIBVERSION=lib_v2.1.14
+LIBVERSION=lib_v2.1.15
```

Comment:
Do we need to do this stuff? Can we just have a single place with version info/modified date? Like, could we have a single file that has all these variables?

Reply:
uh we don't need to do it for each file, no not really.  I am adding the "package version" to connectd_options and that also includes the build date.  So that could satisfy your request, along with removing version info from all files which currently include it.

Reply:
f you like I can handle that as part of https://remoteit.atlassian.net/browse/CTD-301 which puts the package version and build date into a single file.  So I could also remove MODIFIED dates and version numbers from all other files which include them.  There's no rhyme or reason for it other than those things were there when the installer "followed me home" and I've incremented them as I have made changes